test/e2e/aws_test.go

```diff
@@ -133,13 +184,275 @@ var _ = Describe("functional tests", func() {
 			By("Scale the MachineDeployment up again")
 			scaleMachineDeployment(namespace, machineDeploymentName, scaleDownReplicas, initialReplicas)
 
+			By("Creating the LB service")
```

Comment:
Not sure if this should be a part of the `workload cluster lifecycle`

Reply:
The verify, dynamically provisioned ELB deletion and EBS Volumes retention after cluster deletion the ELB and EBS should be created after cluster creation and before cluster deletion. Hence, did it here before cluster deletion.

Reply:
Made this part of another scenario